File: app/src/main/java/cc/smartcash/wallet/Services/TransactionService.java
package cc.smartcash.wallet.Services;

import cc.smartcash.wallet.Models.FullTransaction;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Url;

public interface TransactionService {

    @GET
    Call<FullTransaction> getTransaction(@Url String url);
}

File: app/src/main/java/cc/smartcash/wallet/Services/TransactionService.kt
package cc.smartcash.wallet.Services

import cc.smartcash.wallet.Models.FullTransaction
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Url

interface TransactionService {

    @GET
    fun getTransaction(@Url url: String): Call<FullTransaction>
}
